ui/backend/client_ui.py

```diff
@@ -101,49 +101,3 @@ def check_repo(self, repo_url, git_token=None, local_repo=False):
             except GitCommandError:
                 return False, 'GitCommandError'
         return True, None
-
-    def update_similar_snippets(self,
-                                target_snippet,
-                                state,
-                                repo_url,
-                                file_name=None,
-                                threshold=0.96):
-        """ Find snippets that are similar to the target
-        snippet and update their state.
-
-        Parameters
-        ----------
-        target_snippet: str
-        state: str
-            state to update similar snippets to
-        repo_url: str
-        file_name: str
-            restrict to a given file the search for similar snippets
-        threshold: float
-            update snippets with similarity score above threshold.
-            Values lesser than 0.94 do not generally imply any relevant
-            amount of similarity between snippets, and should
-            therefore not be used.
-
-        Returns
-        -------
-        int
-            The number of similar snippets found and updated
-        """
-
-        discoveries = self.get_discoveries(repo_url,
-                                           file_name,
-                                           state_filter='new')[1]
```

Comment:
now I get why there was this 1 (follow-up from your previous comment in the parent Client class).
So, I'm wondering if we should keep this method here. Or maybe, we can delegate part of this call to Client, and keep in ClientUI the part related to finding the correct discoveries. I can help you on this.

Reply:
yes your help on the matter would be appreciated, thank you, apart from the current one, I can only think of rather complicated solutions